mollie_subscription_ept/static/src/js/cart.js

```diff
@@ -0,0 +1,14 @@
+odoo.define('mollie_subscription_ept.clear_warn', function (require) {
+    "use strict";
+    $('#clear_warn_wishlist_btn').click(function () {
+        var ajax = require('web.ajax');
+        ajax.jsonRpc("/clear_warn", 'call', {'hide_warn':true}).then(function(data) {
+        if (data) {} else {}});
+    });
+
+    $('#clear_warn_in_cart_btn').click(function () {
```

Comment:
As the function body is same for both the click functions , it can be combined into single function and that can be called from each of the click function definition